### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by almereyda** [claudioc/jingo#117]
Looking at the repo's issue activity, it appears to me Jingo has an existing user base and a living developer community.

As there are multiple known use cases and custom adaptations, different approaches in running and utilizing Jingo could be documented in self-contained documentation within the repository itself.

* [ ] If put in the `docs` subfolder, Markdown documentation will easily be rendered by http://progrium.viewdocs.io/viewdocs/ - which also offers some kind of templating one could try to align with the approach here.
* [ ] Hypothes.is and nginx cases could be presented
* [ ] The main README could be trimmed down to administrative essentials, extended by a [CONTRIBUTING.md](https://github.com/blog/1184-contributing-guidelines)

The list above could be exteneded, of course, but may suit as a template for a PR implementing this.

**Comment by claudioc** [comment]
I agree: the README starts to be a bit too big and maybe can be trimmed and the references and use cases moved to the docs folder or the wiki. I think the wiki is better (today I just had to release a version just because I forgot to remove a line from the README. Silly).

I just opened the wiki, in the meanwhile :D

**Comment by claudioc** [comment]
In the meantime: I added a CONTRIBUTING file, and some Wiki pages.

I'd like to remove the settings reference from the README too.

**Comment by philippnagel** [comment]
Wouldn't it be more interesting and appropriate to host the project wiki using Jingo?
